srcpkgs/rofi-emoji/template

```diff
@@ -1,9 +1,9 @@
 # Template file for 'rofi-emoji'
 pkgname=rofi-emoji
-version=2.3.0
+version=3.0.0
 revision=1
 build_style=gnu-configure
-hostmakedepends="autoconf automake libtool pkg-config"
+hostmakedepends="autoconf automake libtool pkgconf"
```

Comment:
why switch?

Reply:
some macro https://github.com/Mange/rofi-emoji/issues/45

Reply:
looks like 3.0.1 fixes it :)

Reply:
Updated to it !